File: Data Structures & Algorithms/minimum-recolors-to-get-k-consecutive-black-blocks/submission-1.java
class Solution {
    public int minimumRecolors(String blocks, int k) {
        int res=0 ;
        for(char c :blocks.toCharArray()){

        }
        for(int i =0 ;i<k;i++){
            if (blocks.charAt(i)=='W'){
                res++;
            }
            
        }
        int ress=res;
        for( int i =k;i<blocks.length();i++){
            if (blocks.charAt(i)=='W'){
                res++;
            }
            if (blocks.charAt(i-k)=='W'){
                res--;
            }
            ress=Math.min(res,ress);
            


        }
        return ress ;
        
    }
}

File: Data Structures & Algorithms/plus-one/submission-1.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        if digits==[9] :
            return [1,0]
        if digits[-1]<9 :
            digits[-1]+=1
            return digits
        else: 
            res= self.plusOne(digits[0:len(digits)-1])
            res.append(0)
            return res
        